_join/index.md

```diff
@@ -47,29 +47,27 @@ We're building a team that reflects the United States. We don't discriminate bas
 
 ### How we collect and review applications
 
-We collect applications during the period specified in the job posting (usually five days to two weeks). We consider all applications collected within a given period a cohort and evaluate them against each other.
+We collect applications during the period specified in the job posting (usually five business days). We consider all applications collected within a given period a cohort and evaluate them against each other.
 
-Once the application period ends, you can't add to or modify your application.
+Once the application collection period ends, you can't add to or modify your application.
 
-We begin evaluating applications using a qualification process called [Category Rating](https://www.opm.gov/policy-data-oversight/hiring-information/competitive-hiring/#url=Category-Rating). Following this process, we evaluate each application against specific competencies (you can find these in the job posting) and categorize each as Superior Qualified, Best Qualified, Well Qualified, Qualified, or Not Qualified.
-
-Once we've evaluated all applications within the cohort, we send all the applications and evaluation scores to General Services Administration (GSA) Human Resources, our partner in the hiring process, to apply [Veterans' Preference](http://www.fedshirevets.gov/job/vetpref/index.aspx) for candidates who claim it. GSA HR returns a list of candidates who were categorized as Superior Qualified, and the selection panel (usually the hiring manager) then identifies which candidates to interview.
+A panel of Subject Matter Experts (SMEs) will review each application. The top qualified candidates will be contacted to continue on to the interview process. 
 
 ----
 
 ## The interview process
 
-The interview process begins with a 30-minute preliminary screening by phone or video call. Plan to talk about your skills and experience, what you're passionate about, and to learn about the work we do here.
+The interview process usually begins with a 30-minute preliminary screening by phone or video call. Plan to talk about your skills and experience and what you're passionate about. We'll also share more about our teams and the work we do here.
 
 If we'd like to chat with you more, we'll invite you to a series of video chats with 18F team members for a couple of longer conversations.
 
-If we agree you're a good fit for the role after interviews, your point of contact on the 18F Talent Team will reach out to you with next steps.
+If a strong match is identified after interviews, your 18F recruiter will reach out to you with next steps.
```

Comment:
The first part of this sentence needs a subject. Does 18F identify them as a strong match or do we want to say if both parties identify each other as a strong match?